src/octree/generation.rs

```diff
@@ -36,7 +37,7 @@ use std::sync::mpsc;
 const UPDATE_COUNT: i64 = 100_000;
 const MAX_POINTS_PER_NODE: i64 = 100_000;
 
-impl CubeNodeWriter {
+impl RawNodeWriter {
```

Comment:
The name might not be descriptive enough now that it's not necessarily "Raw" (unencoded) anymore – how about `PerAttributeNodeWriter`?

Reply:
Yeah, agree, didn't find a better name before... I meant raw in the sense of not having a file format with header... `PerAttribute...` is fine with me, too, just a little long...